### ISSUE TITLE
Particle phantom unclear axis in reconstruction

### ISSUE THREAD

**Issue by Neumann-A** [MagneticParticleImaging/OpenMPIData.jl#10]
So i am currently trying a new custom reconstruction and displaying the data via matlab e.g.: 

```
result = reshape(result,[19 19 19]); % result is the raw reconstruction result ()
imagesc(squeeze(result(13,:,:))) % yz plane
```

So although the xy and xz planes seem to agree the yz plane seems to be transposed? Since the example reconstruction don't have a legend on the axis it is unclear in which direction the corresponding axis is facing and if the third axis is facing inside our outside the plane.

In addition to that I would request that the phantom description gets an additional column saying which voxel the concentration should be reconstructed to. 

Linear scaling to identify concentrations
![Reko-Linear](https://user-images.githubusercontent.com/30894796/219899195-6e4cd55d-ed8a-43a6-8d89-9a99364592bf.png)
Log scaling of the reco to show artifacts and lowest concentration
![Reko-Log](https://user-images.githubusercontent.com/30894796/219899199-83ace419-5221-4361-8cb5-48ae2c821b56.png)

**Comment by tknopp** [comment]
yes, we certainly should add proper axis labels to the example reconstructions. Regarding the actual location of the concentrations I hope that this description is precise:
 https://magneticparticleimaging.github.io/OpenMPIData.jl/latest/phantoms.html

> In addition to that I would request that the phantom description gets an additional column saying which voxel the concentration should be reconstructed to.

I agree that this is useful and in the future we aim to provide ground truths concentration maps. For the cone and the resolution phantom we already have those from the Bremen group (see https://journal.iwmpi.org/index.php/iwmpi/article/view/148/364 where the methodology is explained). For the concentration phantom somebody has to create these. And note that the challenge is that the phantom may cover voxels only partially, i.e. one has to take this into account.
